React/main/src/app/components/CustomInput/styles.scss

```diff
@@ -0,0 +1,44 @@
+.custom-input {
+  --font-size: 10px;
```

Comment:
We used to declare $nameOfVariable: 10px and then write font-size: $nameOfVariable